lib/modules/hosts/derpibooru.js

```diff
@@ -0,0 +1,38 @@
+addLibrary('mediaHosts', 'derpibooru', {
+	name: 'Derpibooru',
+	domains: ['derpiboo.ru'],
+	matchRe: /^https?:\/\/(?:www\.)?derpiboo.ru\/\d+$/i,
```

Comment:
Yeah, I think I'll probably just use [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) then. It's supported in everything but IE.